homeassistant/components/owlet/coordinator.py

```diff
@@ -0,0 +1,63 @@
+"""Owlet integration coordinator class."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from pyowletapi.exceptions import (
+    OwletAuthenticationError,
+    OwletConnectionError,
+    OwletError,
+)
+from pyowletapi.sock import Sock
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OwletCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class OwletCoordinator(DataUpdateCoordinator[None]):
```
`DataUpdateCoordinator` is generic with the return type of `_async_update_data`.